src/Cache/CacheOperationTypes/CacheOperationTypes.php

```diff
@@ -3436,3 +3446,282 @@ class ItemGetTtlError extends ItemGetTtlResponse
 {
     use ErrorBody;
 }
+
+/**
+ * Parent response type for a update ttl request. The
+ * response object is resolved to a type-safe object of one of
+ * the following subtypes:
+ *
+ * * UpdateTtlSet
+ * * UpdateTtlMiss
+ * * UpdateTtlError
+ *
+ * Pattern matching can be used to operate on the appropriate subtype.
+ * For example:
+ * <code>
+ * if ($set = $response->asSet()) {
+ *     // handle set as appropriate
+ * } elseif ($response->asMiss())
+ *     // handle miss as appropriate
+ * } elseif ($error = $response->asError())
+ *     // handle error as appropriate
+ * }
+ * </code>
+ */
+abstract class UpdateTtlResponse extends ResponseBase
+{
+    /**
+     * @return UpdateTtlSet|null Returns the set subtype if the request returned an error and null otherwise.
```

Comment:
```suggestion
     * @return UpdateTtlSet|null Returns the set subtype if the request returned a set and null otherwise.
```
search for "an error and null otherwise" to find the other places.